xml/System.Resources.Extensions/DeserializingResourceReader.xml

```diff
@@ -64,8 +64,8 @@
         <Parameter Name="fileName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="fileName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="fileName">The path and name of the resource file to be read. This parameter is not case-sensitive.</param>
```

Comment:
I'd recommend leaving out `This parameter is not case-sensitive`,  that may be true on windows but not linux and it isn't really a characteristic of this API but the underlying FileSystem APIs.   I know this was copied from ResourceReader docs, maybe we should remove it there too.

Reply:
Ok thanks. Removing it:

```suggestion
        <param name="fileName">The path and name of the resource file to be read.</param>
```